frontend/ci-git-fix.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "git config --global --add safe.directory /app"
+git config --global --add safe.directory /app
```

Comment:
What does this do? Is it for local run or docker?

Reply:
This is a workaround to a reported [bug](https://github.com/actions/runner/issues/2033), it resolves a "dubious" permissions error when running the frontend with docker. We run a git command on frontend startup and this is required to do that.